what_is.md

```diff
@@ -309,9 +316,12 @@ Very brief discussion of efficient counting of topologies, i.e. the combinatoric
 Summary of this subsection:
 
 ```{epigraph}
-Genetic calculations involve iterating over trees, which is highly efficient in ``tskit`` 
+Genetic calculations involve iterating over trees, which is highly efficient in tskit 
 ```
 
+<-- ## An efficient analysis framework (suggested by Ben Haller) -->
```

Comment:
I think we can remove this. We should call the third section "Efficient computation" or something, rather than "analysis", which is too narrow.

Reply:
Ideally, the name should match the title on the [learn.html](https://tskit.dev/learn.html#what) page. Perhaps there we should call it "A framework for efficient computation" instead?

Reply:
Or "efficient calculation" (rather than computation) - not sure which I prefer. Analysis folks might think "calculation" is more applicable to their research than "computation"?